test/machineslib.py

```diff
@@ -85,11 +85,19 @@ def waitNetworkRow(self, networkName, connectionName="system", present="true"):
         else:
             b.wait_not_present(network_row)
 
+    def getLibvirtServiceName(self):
+        m = self.machine
+
+        if m.image in ["fedora-35"]:
+            return "virtqemud"
+        else:
+            return "libvirtd"
```

Comment:
We should invert the condition here. AFAIUI this wouldn't fail on fedora-36, as the monolithic daemon is still a thing, isn't it? If it will fail on F36/RHEL 10/debian-testing etc., then it's ok, and we can invert it once we get more distros with the split daemon.

Reply:
It will fail for sure exactly how fedora-35 tests failed before this PR. 
For the record, the monolithic daemon will be removed much later (fedora 38 they assume)